src/main/java/net/dries007/tfc/network/PacketPlaceBlockSpecial.java

```diff
@@ -46,7 +46,7 @@ public IMessage onMessage(PacketPlaceBlockSpecial message, MessageContext ctx)
                         double placeReach = player.getEntityAttribute(EntityPlayer.REACH_DISTANCE).getAttributeValue();
                         if (player.getDistanceSq(pos) <= placeReach * placeReach && hitFace != null)
                         {
-                            if (!stack.isEmpty() && world.getBlockState(pos).isNormalCube() && world.getBlockState(pos.offset(hitFace)).getBlock().isReplaceable(world, pos))
+                            if (!stack.isEmpty() && world.getBlockState(pos).isNormalCube() && hitFace == EnumFacing.UP && world.getBlockState(pos.up()).getBlock().isReplaceable(world, pos))
```

Comment:
Instead of restricting this to targeting the top block, change this to use `pos.offset(hitFace)` where necessary.